### ISSUE TITLE
Getting errors every time I try now

### ISSUE THREAD

**Issue by FotoCub** [satsukitv/twitter-block-chain#8]
I'm running Blockchain through Chrome on an account I have run it on many times before, but now it is returning an error for every unblocked person. Only trying for the most recent 700-800 accounts.
![untitled](https://user-images.githubusercontent.com/39162772/39875260-bb26a74e-5435-11e8-8a72-bd13f096e08c.jpg)

**Comment by ceceradio** [comment]
hey there! this is a duplicate of #5, please check out that issue for more information (see the bottom 2 posts)
